src/selectors/entities/channel_categories.ts

```diff
@@ -91,36 +92,98 @@ export function makeFilterChannelsByType(): (state: GlobalState, channels: Chann
     );
 }
 
-export function makeFilterAutoclosedDMs(): (state: GlobalState, channels: Channel[], categoryType: string) => Channel[] {
+function getDefaultAutocloseCutoff() {
+    return Date.now() - 7 * 24 * 60 * 60 * 1000;
+}
+
+export function makeFilterAutoclosedDMs(getAutocloseCutoff = getDefaultAutocloseCutoff): (state: GlobalState, channels: Channel[], categoryType: string) => Channel[] {
```

Comment:
Nit: Does is makes sense to change this to be `makeFilterAutoclosedDMsGMs` or something of that sorts?

Reply:
I considered that, but most of the other logic just refers to autoclosing DMs even though it handles both GMs and DMs

Reply:
Never mind. Better to leave it for now IMO too. When i first saw the notification about the review i didnt even realise that it is about GMs too.